content/200-concepts/100-components/02-prisma-client/260-client-extensions/120-query.mdx

```diff
@@ -151,15 +151,16 @@ const xprisma = prisma.$extends({
 
 You can wrap your extended queries into a [batch transaction](/guides/performance-and-optimization/prisma-client-transactions-guide). For example, you can use this to enact row-level security (RLS).
 
-The following example shows how to wrap an extended query into a batch transaction:
+The following example extends `findFirst` so that it runs in a batch transaction.
```

Comment:
Does this make any sense by itself? What is the benefit?

Reply:
@millsp  Can you comment?

Reply:
Current and previous sentence mean the same to me

Reply:
But, does it makes sense to wrap 1 `findFirst` into a batch transaction? Isn't this exactly the same as not wrapping it?

Reply:
OK, I'm going to merge this PR because I'd like to get these improvements into the docs. However, I'll open a separate docs issue to review this example.